docs/pages/jpscore/jpscore_tests_juelich.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Juelich
+keywords: simulation unit test
+tags: [jpscore, test]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_tests_juelich.html
```

Comment:
Is it possible to integrate the results of the juelich tests here automatically?

Reply:
Like rimea, I think it's good to have these results produced on a regular basis, and just link to the results.

Reply:
@chraibi Would you create a new issue?

Reply:
Sure thing!  #632